shogun-config/src/main/resources/application-base.yml

```diff
@@ -91,7 +91,7 @@ spring:
       resourceserver:
         jwt:
           issuer-uri: https://${KEYCLOAK_HOST:shogun-keycloak}/auth/realms/SHOGun
-          jwk-set-uri: https://${KEYCLOAK_HOST:shogun-keycloak}/auth/realms/SHOGun/protocol/openid-connect/certs
+          jwk-set-uri: http://shogun-keycloak:8080/auth/realms/SHOGun/protocol/openid-connect/certs
```

Comment:
Not sure why this is needed, but removing the environment variable `KEYCLOAK_HOST` entirely seems not to be the right way to me.

Reply:
The problem is that spring is accessing the `jwk-set-uri` to autoconfigure itself and retrieve some certificates from keycloak. If the `KEYCLOAK_HOST` is used, this only works if the `KEYCLOAK_HOST` is accessible from inside the container.

Using the container address here seems to be the solution, as this should be working in any environment.

The documentation is not exhaustive in that regard, but I understand it this way:
* The `issuer-url` is used to check the `issuer` field in the jwt. If `jwk-set-uri` is not set, this is also used to autoconfigure and retrieve the certificates.
* The `jwk-set-uri` is used instead for the requests if set

https://docs.spring.io/spring-security/reference/reactive/oauth2/resource-server/jwt.html#webflux-oauth2resourceserver-jwt-jwkseturi